src/_includes/backward-incompatible-changes/commerce/2.3.0-2.4-develop.html

```diff
@@ -426,28 +838,16 @@ <h3 id="2.3.0-2.4-develop-system">System changes</h3>
         <td>System configuration file was added</td>
     </tr>
     <tr>
-        <td>payment/cybersource/webhook_url</td>
-        <td>A field-node was added</td>
-    </tr>
-    <tr>
-        <td>payment/cybersource/is_multidomain</td>
-        <td>A field-node was added</td>
-    </tr>
-    <tr>
-        <td>payment/cybersource/admin_profile_id</td>
-        <td>A field-node was added</td>
-    </tr>
-    <tr>
-        <td>payment/cybersource/admin_access_key</td>
-        <td>A field-node was added</td>
+        <td>system.xml</td>
+        <td>System configuration file was added</td>
     </tr>
     <tr>
-        <td>payment/cybersource/admin_secret_key</td>
-        <td>A field-node was added</td>
+        <td>system.xml</td>
+        <td>System configuration file was added</td>
     </tr>
     <tr>
-        <td>payment/cybersource/webhook_url_admin</td>
-        <td>A field-node was added</td>
+        <td>system.xml</td>
```

Comment:
Duplicate of above?

Reply:
Good catch, thank you.
We will not make any changes in these files until 2.4.0 release, because the content will be rewritten each time the files are generated. The issue must be fixed in a generating tool.

@pdohogne-magento, @jeff-matthews  looks like we have duplicates again in BIC reports.